desktop/src/main/java/bisq/desktop/app/BisqAppMain.java

```diff
@@ -61,7 +61,7 @@ public static void main(String[] args) throws Exception {
 
     @Override
     public void onSetupComplete() {
-        log.debug("onSetupComplete");
+        log.info("onSetupComplete");
```

Comment:
Is this really something we want to see and log already in the info debug level?

Reply:
Good catch. I increased log level to better see this event inside docker. I'll revert it.